indico/modules/rb_new/controllers/backend/bookings.py

```diff
@@ -218,3 +221,15 @@ def _process(self):
         if not self.booking.event.can_access(session.user):
             return jsonify(can_access=False)
         return jsonify(reservation_event_data_schema.dump(self.booking.event).data)
+
+
+class RHMyUpcomingBookings(RHRoomBookingBase):
+    def _process(self):
+        q = ReservationOccurrence.query.filter(
+            ReservationOccurrence.start_dt > now_utc(),
+            db.or_(
+                Reservation.booked_for_user == session.user,
+                Reservation.created_by_user == session.user,
+            )
+        ).join(Reservation).order_by(ReservationOccurrence.start_dt.asc()).limit(5)
```

Comment:
what about using the formatting we use for all other queries?

```python
q = (ReservationOccurrence.query
     .filter(ReservationOccurrence.start_dt > now_utc(),
             db.or_(Reservation.booked_for_user == session.user,
                    Reservation.created_by_user == session.user))
     .join(Reservation)
     .order_by(ReservationOccurrence.start_dt.asc())
     .limit(5)

```

Reply:
`)` missing 😉 
Yeah, you're right!